app/src/main/res/drawable/drawable_button_background_1.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <corners
+        android:bottomLeftRadius="0dp"
+        android:bottomRightRadius="0dp"
+        android:topLeftRadius="8dp"
+        android:topRightRadius="0dp" />
+    <solid android:color="#fff200" />
```

Comment:
TIP: you can also define a file colors.xml under res/values folder and add all custom colors used in app there. A color from colors.xml can be referenced like @color/blue_button_color
and by the way, Android System also have many default colors, those can be referenced, for example @android:color/color_dark_gray.
Notice the difference in referencing @color and @android:color